### DISCUSSION TITLE
compose_mysql

### DISCUSSION THREAD

**Discussion by youngho1234** [GoogleCloudPlatform/microservices-demo#2231] [Q&A]
Hello, I'm a student who recently started studying infrastructure engineering. I came across this demo project and had some difficulties while trying to replace AlloyDB with MySQL in the project. Could you please provide some guidance on how to do this? I would greatly appreciate your help.

**Comment by NimJay** [comment]
@youngho1234 I only _just_ saw this. So sorry (for the late response)!
Tip: For next time, [create a GitHub issue instead](https://github.com/GoogleCloudPlatform/microservices-demo/issues) — you're more likely to receive an immediate response. :)

The only AlloyDB documentation we have for this sample application is [at /kustomize/components/alloydb](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/kustomize/components/alloydb).

Also, if you're looking to move an existing MySQL database into Google Cloud, I suggest looking into [Cloud SQL](https://cloud.google.com/sql?hl=en).
